vmpooler

```diff
@@ -26,4 +26,11 @@ manager = Thread.new {
   ).execute!
 }
 
+if ENV['VMPOOLER_DEBUG']
+  trap("INT") {
+    puts "Shutting down."
+    [api, manager].each { |t| t.exit }
+  }
```

Comment:
Nitpick: multiline blocks should be `do`/`end`, not braces. Not gonna block on this but maybe this repo needs some Rubocop love?